analytics.js

```diff
@@ -5670,8 +5670,7 @@ function object(obj, strict) {
   }
   each(obj, function(key, val) {
     if (isodate.is(val, strict)) {
-      // obj[key] = isodate.parse(val);
-      obj[key] = val;
+      obj[key] = isodate.parse(val);
     } else if (type(val) === 'object' || type(val) === 'array') {
```

Comment:
This change and the one below is related to this [PR](https://github.com/FindHotel/analytics.js/pull/2) I also see you changes analytics.js1 do you know the purpose of this file? 
Let me know if i should make these manual changes as well as on line 5699 below?

Reply:
@jopdeklein makes sense however it seems that the changes applied do not come from `analytics.js-integration-findhotel` instead I traced them back to this
[repo](https://github.com/segmentio/isodate-traverse/blob/1.0.1/lib/index.js) 😬
I guess one good thing is that there was a good reason for you applying this here.

I guess we have a few options:
1. Fork the `isodate-traverse` repo from 1.0.1 and make the customisations on that but not sure which package includes it just now and how we can override it with our repo.
2. Do a manual patch again and document it for next time.
3. Try to update the `isodate-traverse` repo to the latest and see if that fixes what you were trying to do originally (risky option)

Let me know your thoughts I am feeling that option 2 might be best again for now?...